server/src/services/restaurant.ts

```diff
@@ -209,6 +219,7 @@ async function findNearWithinBudget(
   const nearbyQuery = generateNearbyQuery(coordinates, searchRadius);
   const priceFilter = priceFilterQuery(filter?.price_categories);
   const transactionFilter = transactionFilterQuery(filter?.transactions);
+  const foodCategoryFilter = foodCategoryFilterQuery(filter?.food_categories);
   const populateFoodsQuery = generatePopulateFoodsQuery();
 
```

Comment:
some repeat lines here, on you if you feel you want to implement it in a function but I don't think it's necessary

Reply:
😎 

![image](https://user-images.githubusercontent.com/51217000/235284429-01603891-d2c2-44ba-a16c-2695c507fffe.png)